### ISSUE TITLE
Reset After dragging, and when refreshing errors/warnings

### ISSUE THREAD

**Issue by NickHough5** [atlassian/react-beautiful-dnd#1753]
### Expected behavior
To change the order of the items in the list (you can drag and drop then, but it always resets)
And
To not have errors/warnings after refreshing the window

### Actual behavior
When moving an item in the list, it always resets back to its original location
When refreshing the screen it causes the app to give errors (Hard to replicate on CodeSandbox)

### Steps to reproduce
Load https://codesandbox.io/s/failing-rb-dnd-pu8pe
Reorder some items, they always go back to their original place

To get the errors, add a Typeface to the project, this causes an error (unsure if it is the same as the error i'm currently getting) - Also if you edit the code of the ColumnPicker.jsx file this can sometimes also cause an error.

My error/warning show:
Unable to find any drag handles in the context "0"
AND
> Invariant failed: Draggable[id: LastName]: Unable to find drag handle
But again only after refreshing

### Suggested solution?
No idea

### What version of `React` are you using?
16.8.19

### What version of `react-beautiful-dnd` are you running?
12.2.0 (downloaded on the day of this post)

### What browser are you using?
Chrome -> Latest version

### Demo
https://codesandbox.io/s/failing-rb-dnd-pu8pe

P.S sorry if this is a duplicate of other issues raised, i did try search, but came up short

**Comment by alexreardon** [comment]
It looks like you are not setting up keys correctly. Take a look at your `console`

**Comment by NickHough5** [comment]
> It looks like you are not setting up keys correctly. Take a look at your console

Updated the project,
https://codesandbox.io/s/failing-rb-dnd-c1dsc

It states the error occurs with keys
in ErrorBoundary (created by DragDropContext)
in DragDropContext (created by ColumnPicker)

Unless im missing something?

**Comment by NickHough5** [comment]
Update:

To get the error/warnings in my video to stop and to allow dragging/dropping after refreshing i had to do the following

Set the inital state in the constructor to default values (empty ones)
Then in componentDidMount setup the actual values, and set the state from there.

**Comment by danieldelcore** [comment]
Looks like this issue has been resolved 😄 

Will close for now, but feel free to reopen if you're still experiencing this issue
